src/changes-backwards-compatible.apib

```diff
@@ -4,6 +4,10 @@
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
+#### Latest
+
+- We added filtering on a linked `deal` to `events.list`.
```

Comment:
PP: let's make it explicit that the events endpoints now have support for deal links, so we added `link` as a link type plus allow to filter on it.

Reply:
Okay